terraform/cloud-platform/templates/kops.yaml.tpl

```diff
@@ -305,7 +305,14 @@ kind: InstanceGroup
 metadata:
   creationTimestamp: null
   labels:
+    application: justice-cloud-platform
```

Comment:
Didn't really know what to put for application name so came up with this. Any objections?